kubernetes/nodejs-ingress-sidecar/waypoint.hcl

```diff
@@ -0,0 +1,53 @@
+project = "example-nodejs-sidecar"
+app "example-nodejs-ingress-sidecar" {
+ labels = {
+   "service" = "example-nodejs-sidecar",
+   "env"   = "dev"
+ }
+ build {
+  use "pack" {}
+   registry {
+   use "docker" {
+    image = "example-nodejs-sidecar"
+     tag  = "1"
+     local = true
+   }
+  }
+ }
+ config {
+   file = {
+    "/shared-config/nginx.conf" = configdynamic("vault", {
+    path = "secret/data/apps/sample-app"
+     key = "data/nginx.conf"
+    })
+   }
+ }
+ deploy {
+   use "kubernetes" {
+   namespace = "default"
+    probe_path = "/"
+    scratch_path = ["/shared-config"]
+   port {
+    name = "app-port"
+     port = 8080
```

Comment:
We should run a `waypoint fmt` on this file to get it formatted nicely

Reply:
Oh, I didn't know about this! Nice.